src/agentlessd/agentlessd.h

```diff
@@ -20,7 +20,11 @@
 /** Prototypes **/
 
 /* Main monitord */
+#ifdef WAZUH_UNIT_TESTING
+void Agentlessd(void);
+#else
 void Agentlessd(void) __attribute__((noreturn));
+#endif
```

Comment:
Design question: why?

Reply:
the `noreturn` attribute used in a real environment makes the function out an unpredictable result because don't expect to continue running, anyway this trick is used in another UT.